railties/test/generators/app_generator_test.rb

```diff
@@ -954,18 +954,12 @@ def test_skip_webpack_install
       end
     end
 
-    generator([destination_root], skip_webpack_install: true).stub(:rails_command, command_check) do
+    generator([destination_root], skip_webpack_install: true, skip_bundle: true).stub(:rails_command, command_check) do
```

Comment:
The tests for `--skip-javascript` and `--webpack=react` stub a method to avoid running `bundle install`:

https://github.com/rails/rails/blob/60d89c976eeee9f222ace7c6b948e3ea00a64173/railties/test/generators/app_generator_test.rb#L915

https://github.com/rails/rails/blob/60d89c976eeee9f222ace7c6b948e3ea00a64173/railties/test/generators/app_generator_test.rb#L915

Could you do the same here? Mostly just for consistency, but it also makes clear that `skip_bundle: true` isn't part of the command being tested.